main/main.cpp

```diff
@@ -510,6 +515,7 @@ void Main::print_help(const char *p_binary) {
 	OS::get_singleton()->print("  --validate-extension-api <path>   Validate an extension API file dumped (with the option above) from a previous version of the engine to ensure API compatibility. If incompatibilities or errors are detected, the return code will be non zero.\n");
 	OS::get_singleton()->print("  --benchmark                       Benchmark the run time and print it to console.\n");
 	OS::get_singleton()->print("  --benchmark-file <path>           Benchmark the run time and save it to a given file in JSON format. The path should be absolute.\n");
+	OS::get_singleton()->print("  --lsp-port-override <port>        Use the port specified for the language server protocol instead of the one in the editor settings. The port must be between 0 to 65535.\n");
```

Comment:
This section is named "Standalone tools", so that's not the right place for this one (as it doesn't start the LSP, just sets the port).

It should be above in the "Runtime options" list.

Also should be enclosed in the same ifdefs as the code that parses it, so it's not in the help if not actually implemented.

Reply:
Fixed!